jrnl/jrnl.py

```diff
@@ -300,7 +300,11 @@ def _display_search_results(args, journal, **kwargs):
     elif args.export:
         exporter = plugins.get_exporter(args.export)
         print(exporter.export(journal, args.filename))
-
     else:
-        # Default display mode
-        print(journal.pprint())
+        # Display according display_format config option
+        config_selected = kwargs["config"].get("display_format")
```

Comment:
Instead of nesting this conditional in the parent, can you please move it to the top level with the other `elif`s?

Reply:
Hey, thanks that would be better :+1:. Done.